Tests/LibPDF/CMakeLists.txt

```diff
@@ -5,3 +5,6 @@ set(TEST_SOURCES
 foreach(source IN LISTS TEST_SOURCES)
     serenity_test("${source}" LibPDF LIBS LibCore LibPDF)
 endforeach()
+
+file(GLOB PDF_FILES CONFIGURE_DEPENDS "../../Base/home/anon/Documents/pdf/*.pdf")
```

Comment:
I personally would rather list the files explicitly, but this is okay.

Reply:
Done